javascript/linkedList/linkedList/linked-list.js

```diff
@@ -4,92 +4,76 @@ const Node = require("./node");
 
 class LinkedList {
   constructor(value) {
-    this.value = value;
     this.head = null;
   }
 
   isEmpty() {
-    return this.head === null ? true : false;
+    return this.head === null;
   }
 
+  // Add a Node to the front of a linked list
   insert(value) {
-    //insert a node at the beginning of a given LL
-
-    if (this.isEmpty() === true) {
-      // the list is empty
-      // make a new Node with the value given
-      const node = new Node(value);
-      //   // assign this.head to that new node
-      this.head = node;
-    } else if (this.isEmpty() === false) {
```

Comment:
The `if` statement checks if the value is `true` already and, if it's not `true`, it's gonna be `false`. In other words, the explicit `else if` is really just noise so I chose to remove it.